pkg/cmd/pr/merge/merge.go

```diff
@@ -335,21 +336,17 @@ func mergeRun(opts *MergeOptions) error {
 			}
 		}
 
-		localBranchExists := git.HasLocalBranch(pr.HeadRefName)
-		if localBranchExists {
-			err = git.DeleteLocalBranch(pr.HeadRefName)
-			if err != nil {
-				err = fmt.Errorf("failed to delete local branch %s: %w", cs.Cyan(pr.HeadRefName), err)
-				return err
-			}
+		err = git.DeleteLocalBranch(pr.HeadRefName)
+		if err != nil {
```

Comment:
Nit: you can assign and check for error on the same line. I prefer that whenever possible
```suggestion
		if err := git.DeleteLocalBranch(pr.HeadRefName); err != nil {
```